src/components/TallyLoader.js

```diff
@@ -77,11 +77,6 @@ export const TallyLoader = ({ doi, children }) => {
   const [tally, setTally] = useState(null)
   const [notices, setNotices] = useState(null)
   const [error, setError] = useState(null)
-  useEffect(() => {
-    fetchTally({ doi, setTally, setError })
-    fetchNotices({ doi, setNotices, setError })
-  }, [doi])
```

Comment:
Is this intentionally removed?

Reply:
hmmm... no I don't think so not sure what happened there